lib/Sema/CSDiagnostics.h

```diff
@@ -809,8 +823,17 @@ class MissingOptionalUnwrapFailure final : public ContextualFailure {
   }
 
   Type getUnwrappedType() const {
-    return resolveType(getBaseType()->getOptionalObjectType(),
-                       /*reconstituteSugar=*/true);
+    return resolveType(UnwrappedType, /*reconstituteSugar=*/true);
+  }
+
+  std::string getForceUnwrapString() const {
+    std::string forceUnwrapStr;
+    unsigned i = 0;
+    while (i != UnwrapCount) {
```

Comment:
This loop is no-op now, isn't it? The diagnostic returns if number of unwraps is > 1.

Reply:
This is for force unwrap note, the diagnostic note we skip is coalesce one